src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/package.scala

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.interop.backendforfrontend.service
+
+import scala.concurrent.Future
+import it.pagopa.interop.commons.utils.extractHeaders
+
+package object impl {
+  def withHeaders[T](
```

Comment:
this function may have the right to stay in commons

Reply:
it does in fact